css-ecommerce-page.tar/css-ecommerce-page/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header>New Arrivals</header>
+  <div class="row">
+    <div class="column-fifth">
+      <img src="./images/dress1.jpeg" alt="blackdress">
+      <h3>Free People <span class="size">|Size 6</span></h3>
+      <p><span class="sale">$35</span> <span class="price">$125 retail</span></p>
+    <div class="column-fifth">
+      <img src="./images/ysl-bag.jpeg" alt="blackpurse">
+      <h3>Yves Saint Laurent <span class="size">| One-Size</span></h3>
+      <p><span class="sale">$200</span> <span class="price">$1600 retail</span></p>
+    <div class="column-fifth">
+      <img src="./images/key.jpeg" alt="key">
+      <h3>Tiffany <span class="size">| One-Size</span></h3>
+      <p><span class="sale">$476</span> <span class="price">$2000 retail</span></p>
+    <div class="column-fifth">
+      <img src="./images/lv-bag.jpg" alt="lvpurse">
+      <h3>Louis Vuitton <span class="size">| One-Size</span></h3>
+      <p><span class="sale">$95</span> <span class="price">$1200 retail</span></p>
+    <div class="column-fifth">
+      <img src="./images/dress2.jpeg" alt="bluedress">
+      <h3>Reformation <span class="size">| Size 4</span></h3>
+      <p><span class="sale">$20</span>  <span class="price">$198 retail</span></p>
+    </div>
+    </div>
+    </div>
+    </div>
+    </div>
```

Comment:
Pay attention to the hierarchy of your HTML elements! Right now the structure isn't correct, every subsequent `column-fifth` element is a child instead of a sibling... that's why you have so many leftover closing `div` tags here.

These closing `div` tags should be before each other new `<div class="column-fifth">` opening tag.